### PR TITLE
[fe] Package updates

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

This updates packages and makes code updates to the SDK related to it.

Tagging @joyqvq for changes to the secp keypair.

## Test Plan 

Tests should still pass.

### CODE REVIEW

**Comment by Jordan-Mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10174** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/10174" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/10174?utm_source=stack-comment).

File: sdk/typescript/src/cryptography/secp256k1-keypair.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import * as secp from '@noble/secp256k1';
```
Comment (Jordan-Mysten) [inline sdk/typescript/src/cryptography/secp256k1-keypair.ts:4]: @joyqvq this is where the secp changes are.

File: apps/core/tailwind.config.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
-const { fontFamily } = require('tailwindcss/defaultTheme');
-const colors = require('tailwindcss/colors');
+import { type Config } from 'tailwindcss';
+import { fontFamily } from 'tailwindcss/defaultTheme';
+import colors from 'tailwindcss/colors';
 
 /** @type {import('tailwindcss').Config} */
-module.exports = {
+export default {
```
Comment (williamrobertson13) [inline apps/core/tailwind.config.ts:9]: nice

File: apps/explorer/src/components/displaybox/DisplayBox.module.css [hunk 3]
```diff
@@ -8,7 +8,7 @@ img.smallimage {
 
 div.modalcontainer,
 div.detailsbg {
-    @apply fixed top-0 left-0 z-20 h-full w-full;
+    @apply fixed left-0 top-0 z-20 h-full w-full;
```
Comment (williamrobertson13) [inline apps/explorer/src/components/displaybox/DisplayBox.module.css:11]: did you run a formatter for all of these CSS changes, or what's the context here? just curious

**Review by williamrobertson13 — APPROVED** [review]

File: apps/explorer/src/components/displaybox/DisplayBox.module.css (hunk 3 above)
Comment (Jordan-Mysten) [reply apps/explorer/src/components/displaybox/DisplayBox.module.css:11]: Yeah updated prettier so these are just prettier changes

File: apps/core/tailwind.config.ts (hunk 2 above)
Comment (Jordan-Mysten) [reply apps/core/tailwind.config.ts:9]: You love to see it

**Pull request merged**